styles/styles.css

```diff
@@ -122,6 +122,41 @@
     transition: right 0.2s ease-out;
 }
 
+#outline-delay-slider {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    margin: 18px 0;
+    width: 100%;
+}
+
+#outline-delay-slider::-webkit-slider-runnable-track {
+    background: #47484b;
+    border-radius: 1.3px;
+    box-shadow: 1px 1px 1px #000000, 0 0 1px #0d0d0d;
+    cursor: pointer;
+    height: 4px;
+    width: 100%;
+}
+
+#outline-delay-slider::-webkit-slider-thumb {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    background: #368ac7;
+    border-radius: 3px;
+    box-shadow: 1px 1px 1px #000000, 0 0 1px #0d0d0d;
+    cursor: pointer;
+    height: 22px;
+    margin-top: -8px;
+    width: 10px;
+}
+
+#outline-delay-slider:focus {
+  outline: none;
```

Comment:
Please add some focus border/outline to the handle/thumb.
Brackets Button style for comparison (normal and focus):
![2018-11-02_10-48-55](https://user-images.githubusercontent.com/2564094/47931866-44c02c00-de8d-11e8-9143-aaf2edfada1c.png) ![2018-11-02_10-48-42](https://user-images.githubusercontent.com/2564094/47931872-4689ef80-de8d-11e8-8d2e-b373259a5fd8.png)

Reply:
I have not been able to style the handle without doing it with the track as well. And I really dislike that outline on the whole slider when it's focused.

I have removed the entry for focus and I can not do much more. Sorry.